packages/server/src/items/uses/marketstand/createMarket.ts

```diff
@@ -17,6 +17,11 @@ export class CreateMarket implements Use {
   }
 
   interact(mob: Mob, item: Item): boolean {
+    // Find if Fence Gates Are Near Mob In Radius of 1, If Yes Stop Player From Placing Fence To Prevent Pathing Errors
+    if (Item.countTypeOfItemInRadius('gate', mob.position, 1) > 0) {
+      return false;
+    }
```

Comment:
In the future, we should throw up an error message saying we can't do that. Since right now there is no feedback for this failing to place.